carball/analysis/analysis_manager.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, game: Game):
         self.events_creator = EventsCreator(self.id_creator)
         self.should_store_frames = False
         self.df_bytes = None
+        self.force_full = False
```

Comment:
I think this is better as an argument in `create_analysis`. My train of thought was that the meaning of `force_full` was ambiguous as an attribute of the manager and that it should maybe be `force_full_analysis`. But it seems that it's only used in `create_analysis` which has to be called separately anyway and so it can go there as an argument (where it will be clearer what's being forced).

Reply:
I'm going to remove this